### PR TITLE
Fix grab ray pos

### PR DESCRIPTION
**Pull request by MenelvagorMilsom**
## Overview
Moves the grab ray position to the average of the thumb tip and the index tip and fix tests affected by this change.

## Changes
- Fixes: #5450.
- Grab ray position change.
- Change the pinch hand gesture JSON to centre on the new grab point.
- Add a new open hand gesture that is centred on the grab point (mostly for testing)
- Some minor gesture changes to a couple of tests
- Removed NearInteractionTouchableStack test as it was broken (see comments in file)

### CODE REVIEW

File: Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs [hunk 1]
```diff
@@ -245,50 +245,90 @@ private static void TestEvents(TouchEventCatcher[] catchers, int[] eventsStarted
...
+        //   to touchables[3] getting a touch down were no longer occuring, and so the test
+        //   would fail.
+        // - Because stacked touchables are broken, and because this test was only passing
+        //   before because of strange behaviour, I have commented out this test, until we have
+        //   fixed touchables and we have a way to test this properly.
+        // - Here is a link to a PR that aims to fix many aspects of touch, but cannot be
+        //   completed before GA: 
+        //   https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5264 
+
+        ///// <summary>
```
Comment (lukastoenneMS) [inline Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs]: You can also use `#if false .. #endif` to comment this out as one block.

**Comment by MenelvagorMilsom** [comment]
@davidkline-ms  should I retarget this PR to the [2.0.0_stabilization](https://github.com/microsoft/MixedRealityToolkit-Unity/tree/prerelease/2.0.0_stabilization) branch?

**Comment by MenelvagorMilsom** [comment]
run /azp run stabilization_pr

**Pull request merged**